### PR TITLE
Add support for RVA fields to crossgen/R2R

### PR DESCRIPTION
**Pull request by jkotas**
RVA fields are became more common with pre-inititialized ReadOnlySpan<byte>. Fix crossgen to deal with them for R2R.

### CODE REVIEW

File: src/zap/zapinfo.cpp [hunk 1]
```diff
@@ -1715,6 +1715,9 @@ ReadyToRunHelper MapReadyToRunHelper(CorInfoHelpFunc func, bool * pfOptimizeForS
     case corInfoHelpFunc: flags return readyToRunHelper;
 #include "readytorunhelpers.h"
 
+    case CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPEHANDLE:
```
Comment (jkotas) [inline src/zap/zapinfo.cpp]: This change is not related to the RVA field fix. This is a new JIT helper for CoreRT (it is redundant for CoreCLR) that was not getting mapped correctly for R2R.

**Comment by fadimounir** [comment]
LGTM
